test/lib/interpreters/split_clustering_test.rb

```diff
@@ -702,5 +702,34 @@ def test_basic_split
       result = OptimizerWrapper.wrapper_vrp('ortools', { services: { vrp: [:ortools] }}, TestHelper.create(problem), nil)
       assert result
     end
+
+    def test_scheduling_partitions_without_occurency
```

Comment:
What do we mean by `_without_occurency`, maybe `without_lapse` or `without_recurrence` might be better?